main/supervisor/APKBUILD

```diff
@@ -1,14 +1,15 @@
 # Contributor: Peter Bui <pnutzh4x0r@gmail.com>
 # Maintainer:
 pkgname=supervisor
-pkgver=3.3.5
+pkgver=4.0.1
 pkgrel=0
 pkgdesc="A system for controlling process state under UNIX"
 url="http://supervisord.org"
 arch="noarch"
 license="BSD"
-depends="python2 py-meld3 py-setuptools"
+depends="python py-meld3 py-setuptools"
```

Comment:
It does not support `python3`? I think we should explicit depend on the python version number

Reply:
It does, v4 added py3 support.

Reply:
It supports py3 since 4.0 http://supervisord.org/changes.html#id3 that's is why I update dependency as well